### ISSUE TITLE
Accessibility: logs and terminal input are not readable by screen reader, unlabeled buttons on log page

### ISSUE THREAD

**Issue by rfiorentino1** [homebridge/homebridge-config-ui-x#1456]
### Describe The Bug

This is a piggyback off of issue #1226. The screen reader for some reason can't see the text inside of the log field on the status page, or the logs page itself, and same with the terminal input field. Also the 2 buttons on the logs page aren't labeled.

For context, I've been running Homebridge on Mac OS for some time, but I'm about to switch it to a headless version running on a Pi or NAS system, etc. Thus, when I do that, I won't be able to access the terminal directly like I do now from Mac OS, and it would be much easier to do it from the Homebridge UI than having to SSH into the device every time I need to look at a log or update node etc. I've attached [this video](https://www.dropbox.com/s/zxbsrx847f8ubre/Screen%20Recording%202023-02-06%20at%2012.54.42%20PM.mov?dl=0) of what's currently happening if that would be helpful. :)

Thanks so much!

### Homebridge UI Version

4.50.2

### Homebridge Version

1.6.0

### Node.js Version

18.14.0

### Operating System

macOS

### Environment Info

- [ ] Using Docker?
- [ ] Using Hyper-V?
- [X] Using hb-service?

### Raspberry Pi Model

None

**Comment by rfiorentino1** [comment]
Still relevant

**Comment by rfiorentino1** [comment]
Still open

**Comment by rfiorentino1** [comment]
Still open
